modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/ObjectRelationship/ObjectRelationship.tsx

```diff
@@ -16,9 +17,9 @@ import {ReactFieldBase as FieldBase} from 'dynamic-data-mapping-form-field-type'
 import {fetch} from 'frontend-js-web';
 import React, {useEffect, useRef, useState} from 'react';
 
-type LocalizedValue<T> = Liferay.Language.LocalizedValue<T>;
+import {Locale} from '../../../../../../../../../dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/types';
```

Comment:
Hey @devpedrotavares, could you check if you can use:

`import {Locale} from 'dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/types';
`